src/Simulation/Simulators.Tests/CoreTests.cs

```diff
@@ -25,10 +26,27 @@ public CoreTests(ITestOutputHelper output)
             this.output = output;
         }
 
+        [Fact]
+        public void BasicExecution()
+        {
+            var asmPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            var exe = Path.Combine(asmPath, "TestExe", "QsharpExe.exe");
+
+            ProcessRunner.Run(exe, "", out StringBuilder output, out var _, out int exitCode, out Exception ex);
+            Assert.Null(ex);
+            Assert.Equal(1, exitCode);
+            Assert.Contains("NotImplementedException", output.ToString());
```

Comment:
I'm surprised that the exception message goes to standard out. Looking into this more, it seems like there are two exception messages, one that goes to standard out and one that goes to standard error. The one in standard out comes from `SimulatorBase.WriteStackTraceToLog` - all log messages seem to go to standard out.

This is not a big deal and you don't need to change anything here (although I think standard error would work here also, since it goes to both), but just wanted to mention it.

Reply:
Interesting. Either one works for me - I can also check the standard error.